### PR TITLE
Configurable myshopify domain

### PR DESCRIPTION
**Pull request by kmcphillips**
@awd @jamesmacaulay @ilikeorangutans 
cc @haeky @garymardell @shawnfrench 

Uses feature introduced in shopify_app 5.0.2 https://github.com/Shopify/shopify_app/pull/121 to allow us to override the myshopify domain for this app by only adding an extra line to the `shopify_app.yml` file.

Does some simple clean ups for whitespace, quotes, and formatting at the same time.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -55,8 +55,5 @@ class Application < Rails::Application
 
     # Version of your assets, change this if you want to expire all your assets
     config.assets.version = '1.0'
-
-    # Shopify Domain
-    config.shopify_domain = "myshopify.com"
```
Comment (kmcphillips) [inline config/application.rb]: No longer looks here. Now pulls from the `ShopifyApp.configuration` which looks at the `myshopify_domain:` property in the `shopify_app.yml`. cc @haeky

**Comment by awd** [comment]
Awesome - this will make some internal development much easier :smile_cat: :green_heart:

**Comment by garymardell** [comment]
:rocket:

File: config/application.rb (hunk 1 above)
Comment (haeky) [inline config/application.rb]: :+1:

**Pull request merged**